main.fsl

```diff
@@ -0,0 +1 @@
+collection Foo {}
```

Comment:
maybe put this in a test directory? adding `main.fsl` to the project root seems a bit strange